AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -113,7 +116,10 @@ private extension SessionHeaderView {
             }
             
             Button {
-                // action here
+                DispatchQueue.global(qos: .background).async {
```

Comment:
scheduling UI stuff to background queue is not good - changing UI must occur on a main thread due to SDK requirements.